src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -1738,100 +1736,302 @@ <h2 class="text--center">Balance sheet</h2>
 
-                <tr>
-                  <th class = "strong" id="current-assets-heading">Current assets</th>
-                </tr>
+                    <tr>
+                      <th class = "strong" id="current-assets-heading">Current assets</th>
+                    </tr>
 
-                <!-- BALANCE SHEET - CURRENT ASSETS - STOCKS -->
+                    <!-- BALANCE SHEET - CURRENT ASSETS - STOCKS -->
 
-              {{if $stocks := $currentAssets.stocks}}
-                <tr class="figures">
-                  <td id="stocks-label">Stocks:</td>
+                  {{if $stocks := $currentAssets.stocks}}
+                    <tr class="figures">
+                      <td id="stocks-label">Stocks:</td>
 
-                  <td class="figure noteIndex" id="uk-busTotalInventoriesCY_ENDNoteIndexes"></td>
+                      <td class="figure noteIndex" id="uk-busTotalInventoriesCY_ENDNoteIndexes"></td>
 
-                  <td id="stocks-curr-val" class="strong figure">
-                    {{if isNotEmpty $stocks.current_amount}}
-                      <ix:nonFraction contextRef="CY_END" decimals="0" unitRef="GBP" name="core:TotalInventories" format="ixt2:numdotdecimal">{{$stocks.current_amount}}</ix:nonFraction>
-                    {{end}}
-                  </td>
+                      <td id="stocks-curr-val" class="strong figure">
+                        {{if isNotEmpty $stocks.current_amount}}
+                          <ix:nonFraction contextRef="CY_END" decimals="0" unitRef="GBP" name="core:TotalInventories" format="ixt2:numdotdecimal">{{$stocks.current_amount}}</ix:nonFraction>
+                        {{end}}
+                      </td>
 
-                  <td id="stocks-prev-val" class="figure">
-                    {{if isNotEmpty $stocks.previous_amount}}
-                     <ix:nonFraction contextRef="PY_END" decimals="0" unitRef="GBP" name="core:TotalInventories" format="ixt2:numdotdecimal">{{$stocks.previous_amount}}</ix:nonFraction>
-                    {{end}}
-                  </td>
-                </tr>
-              {{end}}
+                      <td id="stocks-prev-val" class="figure">
+                        {{if isNotEmpty $stocks.previous_amount}}
+                         <ix:nonFraction contextRef="PY_END" decimals="0" unitRef="GBP" name="core:TotalInventories" format="ixt2:numdotdecimal">{{$stocks.previous_amount}}</ix:nonFraction>
+                        {{end}}
+                      </td>
+                    </tr>
+                  {{end}}
 
-                <!-- BALANCE SHEET - CURRENT ASSETS - DEBTORS -->
+              <!-- BALANCE SHEET - CURRENT ASSETS - DEBTORS -->
 
-              {{if $debtors := $currentAssets.debtors}}
-                <tr class="figures">
-                  <td id="ac52-label">Debtors:</td>
+                  {{if $debtors := $currentAssets.debtors}}
+                    <tr class="figures">
+                      <td id="debtors-label">Debtors:</td>
```

Comment:
Do we need to match our ids to the ids in the provided SmallFullAccountsModifiediXBRL.html?